src/components/notebook-utils.js

```diff
@@ -73,11 +73,18 @@ export const tools = {
   Jupyter: { label: 'Jupyter', ext: 'ipynb', imageIds: ['terra-jupyter-bioconductor', 'terra-jupyter-bioconductor_legacy', 'terra-jupyter-hail', 'terra-jupyter-python', 'terra-jupyter-gatk', 'Pegasus', 'terra-jupyter-gatk_legacy'], defaultImageId: 'terra-jupyter-gatk' },
   jupyterTerminal: { label: 'terminal' },
   spark: { label: 'spark' },
-  galaxy: { label: 'galaxy' }
+  galaxy: { label: 'galaxy', appType: 'GALAXY' },
+  cromwell: { label: 'cromwell', appType: 'CROMWELL' }
 }
 
 const toolToExtensionMap = { [tools.RStudio.label]: tools.RStudio.ext, [tools.Jupyter.label]: tools.Jupyter.ext }
 
+// Mapping of tool label to appType (or undefined if tool is not an app).
+export const toolToAppTypeMap = { [tools.galaxy.label]: tools.galaxy.appType, [tools.cromwell.label]: tools.cromwell.appType }
```

Comment:
Thought 1: Is there a way of `_.flow` ing this something like:
- Take tools
- Filter by existence of `appType`
- Map to `label -> appType`

Reply:
Thought 2: Why are the keys in this object arrays?

Reply:
Thought 3: Isn't this just a fancy way of rephrasing object lookups? eg aren't these the same:
```
const way1 = toolToAppTypeMap('cromwell')
const way2 = tools.cromwell.Apptype
```